tox.ini

```diff
@@ -163,10 +163,9 @@ deps =
     {[base]deps}
     mypy
     mypy-zope
-    typeshed
```

Comment:
this is no longer needed?

Reply:
typeshed on pypi is super old, and it's now bundled into `mypy` instead of distributed side by side.